src/status_im2/contexts/communities/discover/view.cljs

```diff
@@ -83,18 +81,34 @@
                        colors/neutral-50
                        colors/neutral-40)}]])
 
+(defn discover-communities-segments
+  [selected-tab]
+  [rn/view
+   {:style style/discover-communities-segments}
+   [quo/tabs
+    {:size           32
+     :on-change      #(reset! selected-tab %)
+     :default-active :all
+     :data           [{:id                  :all
+                       :label               (i18n/label :t/all)
+                       :accessibility-label :all-communities-tab}
+                      {:id                  :open
+                       :label               (i18n/label :t/open)
+                       :accessibility-label :open-communities-tab}
+                      {:id                  :gated
+                       :label               (i18n/label :t/gated)
+                       :accessibility-label :gated-communities-tab}]}]])
+
+
 (defn featured-list
   [communities view-type]
   (let [view-size (reagent/atom 0)]
     (fn []
       [rn/view
-       {:style     {:flex-direction :row
-                    :overflow       :hidden
-                    :width          "100%"
-                    :margin-bottom  24}
+       {:style     style/featured-list-container
         :on-layout #(swap! view-size
                       (fn []
-                        (oops/oget % "nativeEvent.layout.width")))}
+                        (- (oops/oget % "nativeEvent.layout.width") 20)))}
```

Comment:
why -20?

Reply:
layout width minus 20 to achieve 

<img width="380" alt="Screenshot 2023-01-27 at 19 13 16" src="https://user-images.githubusercontent.com/28704507/215135687-7651ff4a-f7b3-4851-b66a-113b73172b88.png">

Reply:
Looks great 👍